### ISSUE TITLE
META: Lucas: expiring news or document content feature

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#985]
Lucas asked about the ability to have expiring news content. I asked for the use case and I'm not sold since creating more news items would solve the problem. However, I wanted to get this request in the queue to see if perhaps I'm overlooking the usefulness of the request.

Use case from Lucas:

"The most common case would be to pull it off the home page. It could also be used for seasonal things. I used it a lot for fireworks bans, larger events/programs that span multiple days and have a news item, and news for operational things (budget meetings, elections, town halls, etc.  
Basically anything you do not want to create a page for because it is not permanent, but is something you want to bring to people's attention. It prevents stale data from sitting out there taking up valuable real estate. ":

**Comment by jlyon** [comment]
@kevindherman we could add this with something like https://wordpress.org/plugins/wp-post-expires/

**Comment by jlyon** [comment]
Related: #971 #913

**Comment by kevindherman** [comment]
@jlyon @aschmoe I've added document expiration to the title per our conversation on the huddle.

**Comment by kevindherman** [comment]
@jlyon @aschmoe I'm starting to see this request come up a lot more often. It would be great to get in a release in the near future if possible.
